<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>facteur_inter</class>
 <widget class="QMainWindow" name="facteur_inter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>731</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Reféreance</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_ref">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Montant</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_mont">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajout_button">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>240</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="line_date2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_rem">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Remise</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView222_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>481</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="type_comboBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher </string>
     </attribute>
     <widget class="QTableView" name="table_facteur">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>621</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pardate">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>par date</string>
      </property>
     </widget>
     <widget class="QPushButton" name="partype">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>par type</string>
      </property>
     </widget>
     <widget class="QPushButton" name="actualiser">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>9</y>
        <width>621</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">    background: transparent;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>111.png</normaloff>111.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="exel">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Excel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer Facture</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Référence</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_idsupp">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_supp">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView222_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>481</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="ref_comboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Modifier Facture</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_id_modif">
      <property name="geometry">
       <rect>
        <x>142</x>
        <y>80</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="line_date2_modif">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_ref_modif">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_button">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Montant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_mont_modif">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Reféreance</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_rem_modif">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Remise</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView222">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>481</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="type_comboBox_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clear">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Stat par type de payment</string>
     </attribute>
     <widget class="PieChartWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>9</y>
        <width>701</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="actualiser_stat">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>actualiser</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>calc</string>
     </attribute>
     <widget class="QTableView" name="listView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>521</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ref">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mont">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rem">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>140</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_globe">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>210</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Refereance</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>120</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Montant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>120</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Remise</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>190</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Montant aprés remise (TTC)</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PieChartWidget</class>
   <extends>QWidget</extends>
   <header>piechartwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
